Create 2023-06-30-2-38-0-preview.md (#789)
percona/community: content/blog/2023/2023-06-30-2-38-0-preview.md

---
title: "Percona Monitoring and Management 2.38 preview release"
date: "2023-06-30T00:00:00+00:00"
tags: ['PMM_prev']
authors:
  - taras_kozub
images:
  - superhero.png
slug: preview-release
---

## Percona Monitoring and Management 2.38 preview release

Hello folks! Percona Monitoring and Management (PMM) 2.38 is now available as a preview release.

We encourage you to try this PMM preview release in **testing environments only**, as these packages and images are not fully production-ready. The final version is expected to be released through the standard channels in the coming week.

You can find the Release Notes [here](https://pmm-doc-pr-1081.onrender.com/release-notes/2.38.0.html)

> ### Percona Monitoring and Management server docker installation

Reviewer: ```suggestion
### PMM server Docker installation
```

[Instructions](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/docker.html)

docker tag:

`perconalab/pmm-server:2.38.0-rc`

> **Important:** In order to use the DBaaS functionality during the Percona Monitoring and Management preview release, you should add the following environment variable when starting PMM server:

Reviewer: ```suggestion
**Important:** To use the DBaaS functionality during the Percona Monitoring and Management preview release, add the following environment variable when starting PMM server:
```

> **Important:** In order to use the DBaaS functionality during the Percona Monitoring and Management preview release, you should add the following environment variable when starting PMM server:

Reviewer: ```suggestion
**Important:**To use the DBaaS functionality during the PMM preview release, add the following environment variable when starting PMM Server:
```

`PERCONA_TEST_DBAAS_PMM_CLIENT=perconalab/pmm-client:2.38.0-rc`

> ### Percona Monitoring and Management client package installation

Reviewer: ```suggestion
### PMM client package installation
```

> Download the latest pmm2-client release candidate tarball for 2.38 by this [link](https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/el9/pmm2-client/pmm2-client-latest-5607.tar.gz).

Reviewer: ```suggestion

1. [Download](https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/el9/pmm2-client/pmm2-client-latest-5607.tar.gz) the latest pmm2-client release candidate tarball for 2.38.
```

> If you want to install pmm2-client package, please enable testing repository via Percona-release: 

Reviewer: ```suggestion
2. To install pmm2-client package, enable testing repository via Percona-release: 
```

`
percona-release enable percona testing
`

> install pmm2-client package for your OS via package manager.

Reviewer: ```suggestion
3. Install pmm2-client package for your OS via Package Manager.
```

You can find the Release Notes [here](https://pmm-doc-pr-1081.onrender.com/release-notes/2.38.0.html)

### PMM server Docker installation

[Instructions](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/docker.html)

docker tag:

`perconalab/pmm-server:2.38.0-rc`

**Important:** To use the DBaaS functionality during the Percona Monitoring and Management preview release, add the following environment variable when starting PMM server:

`PERCONA_TEST_DBAAS_PMM_CLIENT=perconalab/pmm-client:2.38.0-rc`

### PMM client package installation

1. [Download](https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/el9/pmm2-client/pmm2-client-latest-5607.tar.gz) the latest pmm2-client release candidate tarball for 2.38.

2. To install pmm2-client package, enable testing repository via Percona-release: 

`
percona-release enable percona testing
`

3. Install pmm2-client package for your OS via Package Manager.

### OVA

[Instructions](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/virtual-appliance.html)

[PMM2-Server-2.38.0.ova](https://percona-vm.s3.amazonaws.com/PMM2-Server-2.38.0.ova)

### AMI

> [Instructions](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/aws.html)

Reviewer: ```suggestion
[Run PMM Server hosted at AWS Marketplace instructions](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/aws.html)
```

`ami-09895e9b605f14cbc`

---

> Contact us in [https://forums.percona.com/](https://forums.percona.com/).

Reviewer: ```suggestion
Contact us on the [Percona Community Forums](https://forums.percona.com/].
```
